### PR TITLE
Enable support for Identity schema

### PR DESCRIPTION
**Pull request by mrsabath**

### CODE REVIEW

File: charts/spire/templates/crd_role_binding.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: spiffe-crd-rolebinding
```
Comment (lumjjb) [inline charts/spire/templates/crd_role_binding.yaml:4]: name change as well to match role name

**Review by lumjjb — APPROVED** [review]
1 minor nit, if not LGTM

File: charts/spire/templates/crd_role.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: spiffe-crd-role
```
[the remark this answers is not in the document]
Comment (mrsabath) [reply charts/spire/templates/crd_role.yaml:7]: I agree with you on this observation, but unfortunately, this is the naming convention initiated by the SPIRE team:
* https://github.com/spiffe/spire/blob/main/support/k8s/k8s-workload-registrar/mode-crd/config/crd_role.yaml#L7
* https://github.com/spiffe/spire/blob/main/support/k8s/k8s-workload-registrar/README.md#crd-mode-configuration
So I didn't want to change too much...

[the remark this answers is not in the document]
Comment (lumjjb) [reply charts/spire/templates/crd_role.yaml:7]: yea that's fine then.

**Pull request merged**